package.json

```diff
@@ -1096,6 +1096,7 @@
     "monaco-editor": "^0.44.0",
     "monaco-yaml": "^5.1.0",
     "mustache": "^2.3.2",
+    "node-diff3": "^3.1.2",
```

Comment:
Would it be possible to use the same library that we use on the FE side for text diffs?
cc @nikitaindik @jpdjere

Reply:
Both of the diffing-related libraries we use on the front end don't support 3-way diffs, just 2-way. I did try to use existing packages before adding a new one

Reply:
Yep, I can confirm both the `diff` lib and the `diff-match-patch` are for 2-way diffs only.

Reply:
Thank you